timeplus/client.go

```diff
@@ -266,15 +269,15 @@ func (s *TimeplusClient) InsertData(data *IngestPayload) error {
 	return nil
 }
 
-func (s *TimeplusClient) QueryStream(sql string) (rxgo.Observable, *QueryInfo, error) {
+func (s *TimeplusClient) QueryStreamV1(sql string) (rxgo.Observable, *QueryInfo, error) {
```

Comment:
Let's simply remove this completely? Is there any reason we still want to have V1?